community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ImportLogic.java

```diff
@@ -510,7 +510,8 @@ public void close() throws IOException
     {
         // We're done, do some final logging about it
         long totalTimeMillis = currentTimeMillis() - startTime;
-        String additionalInformation = getState( DataStatistics.class ).toString();
+        DataStatistics state = getState( DataStatistics.class );
+        String additionalInformation = Objects.toString( state, "Data diagnostics is not available." );
```

Comment:
The term `diagnostics` has more of a medical/repair/investigation meaning, hasn't it. `Data statistics` may look better imho.